usmqe_tests/tendrlapi/test_volume.py

```diff
@@ -0,0 +1,268 @@
+"""
+REST API test suite - gluster volume
+"""
+import pytest
+
+from usmqe.api.tendrlapi import tendrlapi
+from usmqe.gluster import gluster
+from usmqe.api.etcdapi import etcdapi
+import usmqe.inventory as inventory
+
+
+@pytest.fixture
+def cluster_id():
+    api = tendrlapi.ApiGluster()
+    return api.get_cluster_list()[0]["cluster_id"]
+
+
+@pytest.fixture
+def volume_id():
+    test_gluster = gluster.GlusterCommon()
+    xml = test_gluster.run_on_node(command="volume info")
+    return xml.findtext("./volInfo/volumes/volume/id")
+
+
+LOGGER = pytest.get_logger('volume_test', module=True)
+"""@pylatest default
+Setup
+=====
+
+Further mentioned ``APIURL`` points to: ``http://USMSERVER:8080``.
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+"""@pylatest api/gluster.cluster_import
+API-gluster: cluster_import
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Import gluster cluster.
+"""
+
+
+def test_cluster_import():
```

Comment:
I  think this test should be in different file because all other tests here are about volumes